share/functions/fish_vi_cursor.fish

```diff
@@ -10,51 +10,51 @@ function fish_vi_cursor -d 'Set cursor shape for different vi modes'
         return
     end
 
-    # Emacs Makes All Cursors Suck
-    if set -q INSIDE_EMACS
-        return
-    end
+    # If this variable is set, skip all checks
+    if test -z "$fish_force_vi_cursor"
```

Comment:
We usually use something like `not set -q fish_force_vi_cursor`, like for the git prompt variables.

Reply:
Done.